components/Header/index.tsx

```diff
@@ -3,16 +3,15 @@ import React from 'react'
 
 const Header = ({ user }: any) => (
   <AppBar position="static">
-    <div className="bg-secondary pl-10  p-5 flex justify-between">
+    <div className="bg-secondary pl-10  p-5 h-26 flex justify-between">
       <img src="./images/wallora-logo-170x53.jpeg" alt="This is Wallora logo." />
-      <span className="text-primary">This is header</span>
       {user ? (
-        <Button variant="contained" color="primary" href="/api/auth/logout">
-          LOGOUT
+        <Button variant="contained" color="primary" className="h-12" href="/api/auth/logout">
```

Comment:
If you look at the button dimension it is taking height as h-26 and not h-12. That's happening because it is created using <a> tag and not <button>. 
![image](https://user-images.githubusercontent.com/195256/120014056-6ecf2180-bf96-11eb-84db-8ca7df2f2dd5.png)
And hence it is not a block element to retain its own height
The buttons created in the samples [here](https://material-ui.com/components/buttons/) are of type <button> 
Figure out why these sample buttons are not showing up in this component

Reply:
Updated to add button inside div and added block on them to retain size.